packages/calypso-products/src/constants/jetpack.ts

```diff
@@ -23,6 +23,7 @@ export const PRODUCT_JETPACK_CRM = 'jetpack_crm';
 export const PRODUCT_JETPACK_CRM_MONTHLY = 'jetpack_crm_monthly';
 export const PRODUCT_JETPACK_CRM_FREE = 'jetpack_crm_free';
 export const PRODUCT_JETPACK_CRM_FREE_MONTHLY = 'jetpack_crm_free_monthly';
+export const PRODUCT_YOAST_PREMIUM = 'yoast_premium';
```

Comment:
Why is this a Jetpack constant?

Reply:
why not? 😅 changed here https://github.com/Automattic/wp-calypso/pull/53126/commits/39d07164130b2c27837395389b4c8198ade9d6bc